### ISSUE TITLE
PlayStation Home offline build - keyboard is WAY too sensitive

### ISSUE THREAD

**Issue by SlyCooperReloadCoded** [RPCS3/rpcs3#10901]
## Details

Keyboard emulation in the offline build of PlayStation Home is WAY too sensitive.  Pressing any key once registers anywhere from 3-5 key presses.  It doesn't have the constraints of a normal input screen like you'd find on a PC, it's way too sensitive.  Technically you can make it register only one key press at a time, but you have to be so quick to press the key that it's not worth trying at all.  The recent keyboard update seems to have changed this slightly, but still not nearly enough.  For example, if I hold down the A key in this bug report, there's a small delay before it starts spamming A.  Home does not have this delay and will immediately start spamming.

#### 1. Please attach RPCS3's log.

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/7193134/RPCS3.log.gz)

#### 2. Please attach screenshots of your problem.

In this screenshot, I press the I key 10 times.  It registers a lot more than 10 I key presses.  The issue also happens with the game's chat features, but it would look exactly the same as this.

![Capture](https://user-images.githubusercontent.com/58268914/133946852-b025d43c-3180-486c-b201-cf38e7d839e7.PNG)

#### 3. Please provide comparison with real PS3.

This problem does not exist on real consoles, it's only on RPCS3.  Everyone in the Destination Home community has found the same results.

#### 4. Please provide your system configuration:
* OS - Windows 10 x64 19043
* CPU - Intel Core i9-10900K
* GPU - NVIDIA RTX 2080 Super
* Driver version - 466.27.0.0
* RAM - 64 GB DDR4

**Comment by Asinin3** [comment]
What fps are you at when the native keyboard window is open? Do you have fps capped?

**Comment by SlyCooperReloadCoded** [comment]
Yes, FPS is capped at 30, it's native 30 FPS but some elements will go higher if not capped at 30.

**Comment by Megamouse** [comment]
does it do the key auto-repeat on ps3 as well (slower of course)?

**Comment by SlyCooperReloadCoded** [comment]
Yes, I believe so, but it takes the same amount of time as a Windows keyboard.

**Comment by Megamouse** [comment]
Do you use a keyboard as pad? If so, then you should either try with a controller or disable the keyboard in the IO settings of the regular settings dialog.
Unless you already did that.

**Comment by SlyCooperReloadCoded** [comment]
No, I set Keyboard Hander to Basic and use that.

**Comment by SlyCooperReloadCoded** [comment]
Sorry for the above comment, I had just woken up and wasn't feeling well.  Let me explain better:

We all use controllers for general game input (DS4/DS3/Xbox) but the Home offline build requires a USB keyboard be plugged into the console, or in the case of RPCS3, "Keyboard Handler" set to "Basic".  The problem isn't pad sensitivity.

**Comment by Megamouse** [comment]
It might either be a HLE cellKb issue, some spu inaccuracy or similar, or some issue with how the game peceives time.

It's probably worth playing around with cellKb.
I'll try that if I get my hands on the game.
